services/web/server/tests/sandbox/projects_openapi_generator.py

```diff
@@ -55,10 +138,39 @@ class Node(BaseModel):
     # var outputs?
 
 
+# --------------
 class Project(BaseModel):
+    """Domain model"""
+
     id: UUID
     pipeline: Dict[UUID, Node]
```

Comment:
use NodeID

Reply:
these are not that important since I only use them for prototyping

Reply:
yes I know... but expecting they come in one day